src/test/java/api/support/matchers/EventMatcher.java

```diff
@@ -0,0 +1,71 @@
+package api.support.matchers;
+
+import static com.jayway.jsonpath.matchers.JsonPathMatchers.hasJsonPath;
+import static org.folio.circulation.support.JsonPropertyFetcher.getBooleanProperty;
+import static org.hamcrest.core.Is.is;
+import org.hamcrest.Matcher;
+
+import org.folio.circulation.domain.representations.LoanProperties;
+import org.folio.circulation.services.EventPublishingService;
+
+import io.vertx.core.json.JsonObject;
+
+public class EventMatcher {
+
+  public static Matcher<JsonObject> isCheckedOutEvent() {
+    return JsonObjectMatcher.allOfPaths(
+      hasJsonPath("eventType", is(EventPublishingService.ITEM_CHECKED_OUT_EVENT_TYPE))
```

Comment:
I don't think the API tests should depend upon the production code directly. 

What if the name in the constant was incorrect? 

I think this tends to create the opportunity for false positives and reduce awareness of the impact of changes.

Reply:
Fixed